hw/ip/hmac/data/hmac.prj.hjson

```diff
@@ -4,6 +4,9 @@
 
 {
     name:               "hmac",
+    design_spec:        "hw/ip/hmac/doc",
+    dv_plan:            "hw/ip/hmac/doc/dv_plan",
+    checklist:          "hw/ip/hmac/doc/checklist",
```

Comment:
we may have to document this new format here as well:
https://docs.opentitan.org/doc/project/hw_stages/
in the section called "Indicating Stages and Making Transitions"

Reply:
Done, thanks!